Makefile

```diff
@@ -1,8 +0,0 @@
-all: deps
```

Comment:
Does removing the `Makefile` entirely break any build scripts?

Reply:
The makefile was gx rewriting so anything using it would already have been broken.